docs/Guide/getting-started/creating-a-basic-app-command.mdx

```diff
@@ -54,15 +54,14 @@ export class PingCommand extends Command {
   }
 
   public override async chatInputRun(interaction: Command.ChatInputCommandInteraction) {
-    const msg = await interaction.reply({ content: `Ping?`, ephemeral: true, fetchReply: true });
+    await interaction.reply({ content: `Ping?`, flags: 'Ephemeral' });
```

Comment:
lets use the MessageFlags enum instead!

Reply:
Yes, you are totally right. But I spent 30 minutes yesterday looking for it, and I couldn't find it. I'll update it.